tests/test_sqlcompletion.py

```diff
@@ -4,10 +4,17 @@
 from pgcli.packages.parseutils.tables import TableReference
 import pytest
 
-# Returns the expected select-clause suggestions for a single-table select
+
+def table_reference(table, schema=None, alias=None, is_function=False):
```

Comment:
`TableReference` instantiation is already a one-liner. What's the point of wrapping it into a function?